bot/exts/evergreen/wikipedia.py

```diff
@@ -0,0 +1,109 @@
+import asyncio
+import datetime
+import logging
+from typing import List, Optional
+
+from discord import Color, Embed, Message
+from discord.ext import commands
+
+from bot.constants import Wikipedia
+
+log = logging.getLogger(__name__)
+
+SEARCH_API = "https://en.wikipedia.org/w/api.php?action=query&list=search&srsearch={search_term}&format=json"
+WIKIPEDIA_URL = "https://en.wikipedia.org/wiki/{title}"
+
+
+class WikipediaCog(commands.Cog):
+    """Get info from wikipedia."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+        self.http_session = bot.http_session
+
+    @staticmethod
+    def formatted_wiki_url(index: int, titles: str) -> str:
+        """Making formatted wikipedia link.."""
+        return f'`{index}` [{titles}]({WIKIPEDIA_URL.format(title=titles.replace(" ", "_"))})'
```

Comment:
```suggestion
    def formatted_wiki_url(index: int, title: str) -> str:
        """Making formatted wikipedia link.."""
        return f'`{index}` [{title}]({WIKIPEDIA_URL.format(title=title.replace(" ", "_"))})'
```
Titles is still plural